doc/adrs/137_use_a_common_identification_strategy_for_transient_representations.adoc

```diff
@@ -0,0 +1,32 @@
+= [ADR-137] Use a common identification strategy for transient representations
+
+== Context
+
+A representation identifier must be any kind of string.
+Persisted representations are using a `UUID` which is used as the primary key of the representation in Sirius Web.
+Transient representations have various kind of id from `UUID` to various string based identifiers.
+
+The issue is that we need to know that a representation is transient or not in order to look for its metadata in the database or in memory.
+
+== Decision
+
+All transient representations will switch to a URI based identifier with a prefix defining the kind of representation.
+This will change the id of the following representations to:
+
+- Explorer: `explorer://?expandedIds=[]&activeFilterIds=[]`
+- Details: `details://?objectIds=[]`
+- Validation: `validation://`
+- RelatedElements: `relatedElements://?objectIds=[]`
+- Representations: `representations://?objectIds=[]`
```

Comment:
Also `selection://` from `sirius-components-selection`?

Reply:
Yes indeed, I'll add it